### PR TITLE
[release-4.11] infra: Fix SRO deployment

### PR DESCRIPTION
**Pull request by SchSeba**
Signed-off-by: Sebastian Sch <sebassch@gmail.com>

### CODE REVIEW

File: hack/setup-build-index-image.sh [hunk 1]
```diff
@@ -72,6 +72,7 @@ spec:
 
           git clone --single-branch --branch OPERATOR_RELEASES https://github.com/openshift/special-resource-operator.git
           cd special-resource-operator/bundle/OPERATOR_VERSION/
+	  rm manifests/image-references
```
Comment (cgoncalves) [inline hack/setup-build-index-image.sh:75]: Is this a file or directory? If a directory, it will require -r parameter

Comment (SchSeba) [reply hack/setup-build-index-image.sh:75]: it's a file this is just a backport :)

**Comment by mgourin** [comment]
@SchSeba All the failing tests seem to stem from a faulty YAML configuration:
error: error parsing STDIN: error converting YAML to JSON: yaml: line 40: found a tab character where an indentation space is expected

**Pull request merged**